src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.c

```diff
@@ -4,6 +4,16 @@
 
 #include "pal_ssl.h"
 
+// This would come from private header  <Security/SecureTransportPriv.h>
+// 10.13.4 introduced public API but linking would fail on all prior versions.
```

Comment:
We don't call private API.

If this is lightup, though, that's fine.  (And if the lightup happens to unexpectedly work on earlier versions, that's fine, too).

If it's public API in newer versions than our build machine, that's just lightup.  (And the comment should be adjusted accordingly)